internal/handlers/iso_test.go

```diff
@@ -303,6 +346,82 @@ var _ = Describe("ServeHTTP", func() {
 			expectSuccessfulResponse(resp, []byte("someisocontent"))
 		})
 
+		It("passes non empty kargs", func() {
```

Comment:
I think a lot of the setup in these tests would be done for you if you put it in the `with no auth` context, right?

Plus it would make sense since you're not specifically testing auth here.

Reply:
With the test `fail kargs infra env query` it is OK, but with the test `passes non empty kargs` I needed to change the `mockImageStream` function.